### DISCUSSION TITLE
naming convention distinction between autora and firebase experiment

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#365] [Core Contributions]
I am currently working on a tutorial for the closed loop with firebase, and I want to incorporate the suggestion made yesterday of organizing the autora-python-scripts and JavaScript code into one project, with each set of files placed in a separate folder. Can you suggest names for these folders? The Autora scripts are responsible for running the experimentalist, runner, and theorist, while the JavaScript code sets up the experiment on Firebase. I am considering using 'test-site' to describe the folder containing the JavaScript code and "office" for the autora folder, but I am unsure, especially for the folder containing the Autora scripts.

**Comment by musslick** [comment]
Hmm, I would use something slightly more descriptive, like "**test-experiment**" since "test-site" could also be mistaken to refer to a documentation site. As for the autora folder, I wold suggest "**autora-workflow**" or "**test-workflow**" to conform with our new terminology.

**Comment by benwandrew** [comment]
`researcher_suite`? `researcher_station`? `researcher_hub`? `cockpit`?

**Reply by benwandrew** [reply]
i sort of like some variation of `cockpit` or `hub` for the autora folder

**Reply by younesStrittmatter** [reply]
yes, I like that, too! I like researcher_hub.
